integreat_cms/cms/linklists.py

```diff
@@ -1,10 +1,18 @@
 from django.conf import settings
+from django.db.models import Exists, OuterRef
 from linkcheck import Linklist
 
-from .models import Event, EventTranslation, Page, PageTranslation, POITranslation
+from .models import (
+    Event,
+    EventTranslation,
+    LanguageTreeNode,
+    Page,
+    PageTranslation,
+    POITranslation,
+)
 
 
-class ContentTranslationLinklist(Linklist):
+class ActiveLanguageLink(Linklist):
```

Comment:
```suggestion
class ActiveLanguageLinklist(Linklist):
```
I think it makes sense to point out that this class is a link list, not just a link.